.azure_pipelines/job_templates/olive-example-template.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
         scriptLocation: 'inlineScript'
         inlineScript: |
           python -m pip install pytest
+          python -m pip install azure-storage-blob
```

Comment:
should this line be in requirements.txt?

Reply:
It is in requirements.txt before. But the azure-storage-blob is only used for e2e tests to download requirement sdk/models etc. When user run it locally, it is not needed. So I tried to move it here.